website/cue/reference/components.cue

```diff
@@ -1,5 +1,52 @@
 package metadata
 
+#BytesReceivedEvent: {
+	emits_event:               bool
+	has_bytes_size_property:   bool
+	has_protocol_property:     bool | null
+	has_http_path_property:    bool | null
+	has_socket_property:       bool | null
```

Comment:
I might group this like:

```
has_properties: {
  byte_size: bool
  protocol: bool | null
  http_path: bool_null
}
```

Just to avoid putting `has_*_property` a bunch of times. Not a big deal though.